ui/core/src/model/annotations.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Definition } from './definitions';
+import { Display } from './display';
+
+export type AnnotationEvent = {
+  timestamp: number;
+  endTimestamp?: number;
```

Comment:
Will they always be Unix timestamps--I guess, if not, it would have been converted already in the plugin?

Also, is the equivalent for Grafana, just 'from' and 'to'? If so, another option is 'start' and 'end' kind of like how we did the AbsoluteTimeRange query params in TimeRangeProvider

Reply:
https://github.com/sampoder/extensions/blob/e9290fd962be2392489505b76e42558e12683145/extensions/grafana/src/components/annotation/interface/index.ts#L1-L4

Grafana uses `timeEnd` - happy to change this as well. Although `start` `end` is a little vague without the context of time. I was going to use unix ms, so maybe a type to make that more clear.